### PR TITLE
Update PID Profile

### PR DESCRIPTION
**Pull request by haslinghuis**
Changes for https://github.com/betaflight/betaflight/pull/11779

### CODE REVIEW

File: src/SCRIPTS/BF/PAGES/pid_advanced.lua [hunk 1]
```diff
@@ -33,6 +33,12 @@ if apiVersion >= 1.044 then
     fields[#fields + 1] = { t = "Thrust Linear",     x = x,          y = inc.y(lineSpacing), sp = x + sp, min = 0, max = 150, vals = { 57 } }
 end
 
+if apiVersion >= 10.45 then
```
Comment (klutvott123) [inline src/SCRIPTS/BF/PAGES/pid_advanced.lua:36]: Should be 1.045

File: src/SCRIPTS/BF/PAGES/rates.lua [hunk 2]
```diff
@@ -75,7 +75,7 @@ if apiVersion >= 1.016 then
     end
 end
 
-if apiVersion >= 1.016 then
+if apiVersion >= 1.016 and apiVersion <= 10.44 then
```
Comment (klutvott123) [inline src/SCRIPTS/BF/PAGES/rates.lua:78]: Should be 1.044. We should change the api version calculation to get rid of the zero before minor version, but that's for another PR 🙂

**Review by klutvott123 — CHANGES_REQUESTED** [review]

File: src/SCRIPTS/BF/PAGES/pid_advanced.lua (hunk 1 above)
Comment (haslinghuis) [reply src/SCRIPTS/BF/PAGES/pid_advanced.lua:36]: Fixed.

File: src/SCRIPTS/BF/PAGES/rates.lua (hunk 2 above)
Comment (haslinghuis) [reply src/SCRIPTS/BF/PAGES/rates.lua:78]: Fixed. Yes good call as this was programmer blindness ;-)

**Review by klutvott123 — APPROVED** [review]

**Comment by haslinghuis** [comment]
Rebased

**Review by klutvott123 — APPROVED** [review]

**Pull request merged**